style.css

```diff
@@ -0,0 +1,1498 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "ARS Maquette", sans-serif;
+  src: url(assets/font/ARSMaquettePro-Regular.ttf);
+}
+
+@font-face {
+  font-family: "Avalon";
+  src: url(assets/font/Avalon-Bold.ttf);
+}
+
+@font-face {
+  font-family: "Shentox", sans-serif;
+  src: url(assets/font/Shentox.ttf);
+}
+
+@font-face {
+  font-family: "ARS Maquette-bold";
+  src: url(assets/font/ARSMaquettePro-Bold.ttf);
+}
+
+body {
+  margin: 0 auto;
+  font-family: "ARS Maquette", sans-serif;
+}
+
+header a {
+  text-decoration: none;
+}
+
+/* Generic Classes */
+
+#test {
+  overflow: hidden;
+  text-indent: 100%;
+  white-space: nowrap;
+}
+
+.test {
+  overflow: hidden;
+  text-indent: 100%;
+  white-space: nowrap;
+}
+
+.white-text {
+  color: white;
+}
+
+.strikethrough {
+  text-decoration: line-through;
+}
+
+.gray-text {
+  color: #fff9;
+}
+
+.light-gray-text {
+  color: #aaa;
+}
+
+.uppercase {
+  text-transform: uppercase;
+}
+
+.smaller {
+  font-size: 13px;
+}
+
+.small {
+  font-size: 14px;
+}
+
+.gray {
+  background-color: gray;
+}
+
+.mr-30 {
+  margin-right: 30px;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.justify-content-center {
+  justify-content: center;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.align-items-baseline {
+  align-items: baseline;
+}
+
+.align-items-flex-end {
+  align-items: flex-end;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.space-around {
+  justify-content: space-around;
+}
+
+.flex-end {
+  justify-content: flex-end;
+}
+
+.grid {
+  display: grid;
+}
+
+.bigger {
+  font-size: 56px;
+}
+
+/* Nav Section */
+
+.hamburger-icon-mobile {
+  display: none;
+}
+
+.search-icon-mobile {
+  display: none;
+}
+
+.tablet-tnw-logo {
+  display: none;
+}
+
+.latest-funding a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.footer-links-social a {
+  background-color: #333;
+  padding: 12px;
+  border-radius: 50%;
+  margin: 10px;
+  display: flex;
+}
+
+.footer-links-menu a {
+  color: #aaa;
+  margin: 20px 6px;
+  text-decoration: none;
+  font-size: 12px;
+  font-family: "Shentox", serif;
+}
+
+.latest-news h2 a {
+  text-decoration: none;
+  font-family: "Avalon", serif;
+  font-size: 28px;
+  color: #f42;
+}
+
+.card-1-mobile-text h3 a {
+  color: black;
+  text-decoration: none;
+}
+
+.latest-funding h2 a {
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ]  Please add a class to `<a>` element` and delete the left unnecessary rules

**Example:**
```CSS
/* before */
.card-1-mobile-text h3 a {

/* after */
.anchortag-class {
```
<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_